Sortable.js

```diff
@@ -255,7 +255,7 @@
 
 		// Bind all private methods
 		for (var fn in this) {
-			if (fn.charAt(0) === '_') {
+			if (fn.charAt(0) === '_' && typeof(this[fn].bind) === 'function') {
```

Comment:
Maybe, but imagine someone had `_myObj = {bind: 'notafunction'}`, you'd end up with error again...